src/utils/createAjv.ts

```diff
@@ -0,0 +1,30 @@
+import AJV from 'ajv';
+import addFormats from 'ajv-formats';
+import ajvErrors from 'ajv-errors';
+
+let cached: AJV;
+
+export default function () {
+	if (cached) return cached;
+
+	const ajv = new AJV({
+		allErrors: true,
+		verbose: true,
+		strict: false,
+	});
+
+	addFormats(ajv);
+
+	ajv.addKeyword({
+		keyword: 'bmi-isNotEmpty',
+		type: 'string',
+		validate: function (schema: any, data: any) {
```

Comment:
should the 'schema' argument be removed, because it is not used?

Reply:
We need the 2nd argument from this callback which signature is determined by the ajv package. So there's no way to omit the first argument.